src/sharebuttonview.js

```diff
@@ -50,29 +48,31 @@ class ShareButtonView {
       + "/sheet/" + SheetID + "/state/analysis/options/clearselections";
   }
 
+  handleClick() {
+    if (!this.isInEdit) {
+      //var self = this;
+      this.setAndCopyUrl(this.selectionUrl);
+      this.showSuccess();
+      window.onbeforeunload = null;
+      return false;
+    }
+  }
+
   //Updates the button state based on the current component state.
   updateButtonState() {
-    let buttonId = `#${this.id}-generateDashboardLink`;
-    let button = $(`${buttonId}`);
-    button.parent().off(`click.${LISTENER_NAMESPACE}`, `${buttonId}:enabled`);
-
+    var button = document.getElementById(`${this.id}-generateDashboardLink`);
```

Comment:
consider using
```suggestion
    var button = $(document.getElementById(`${this.id}-generateDashboardLink`));
```
to avoid having to change the rest of this method

Reply:
Yes, that's a better replacement. I shall change this.